package.json

```diff
@@ -25,6 +25,7 @@
     "test:watch": "jest --watch --onlyChanged --verbose true"
   },
   "devDependencies": {
+    "@amsterdam/asc-assets": "^0.29.3",
```

Comment:
This dependency should not be added here. The packages should be linked by NPM automatically.

Reply:
weg

Reply:
weg